src/main/scala/djinni/generator.scala

```diff
@@ -744,16 +788,17 @@ abstract class Generator(spec: Spec) {
   def writeEnumOptionAll(
       w: IndentWriter,
       e: Enum,
-      ident: IdentConverter
+      ident: IdentConverter,
+      delim: String = "="
   ): Unit = {
     for (
       o <- e.options.find(_.specialFlag.contains(Enum.SpecialFlag.AllFlags))
     ) {
       writeDoc(w, o.doc)
-      w.w(ident(o.ident.name) + " = ")
+      w.w(ident(o.ident.name) + s" $delim ")
       w.w(
-        normalEnumOptions(e)
-          .map(o => ident(o.ident.name))
+        normalEnumOptions(e).zipWithIndex
+          .map { case (o, i) => s"(1 << $i)" }
```

Comment:
needs a closer look, 
We write now default values to enums. This changes expected outcome, and the reason might be here